packages/notifications/src/inAppMessaging/errors/validation.ts

```diff
@@ -24,4 +25,9 @@ export const validationErrorMap: AmplifyErrorMap<InAppMessagingValidationErrorCo
 		[InAppMessagingValidationErrorCode.NoEndpointId]: {
 			message: 'Could not find or create EndpointId.',
 		},
+		[InAppMessagingValidationErrorCode.NotInitialized]: {
+			message: 'InAppMessaging not initialized.',
+			recoverySuggestion:
+				'Make sure to call initializeInAppMessaging API first.',
```

Comment:
Just wanted to align some with PN

```suggestion
			message: 'In-app messaging has not been initialized.',
			recoverySuggestion:
				'Please make sure to first call `initializePushNotifications`.',
```